build-system/single-pass.js

```diff
@@ -608,6 +605,9 @@ function formatSinglePassClosureCompilerError(message) {
 }
 
 function compile(flagsArray) {
+  // Override to local closure compiler JAR
+  ClosureCompiler.JAR_PATH = require.resolve('./runner/dist/runner.jar');
```

Comment:
what was the reason to move?

Reply:
Single pass was setting the jar at the global level. I moved it to local so it doesn't leak across `gulp` tasks.